packages/stark-ui/src/modules/session-ui/pages/login/login-page.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="stark-session-ui-page"
+	 role="alertdialog"
+	 aria-busy="true"
+	 aria-live="assertive"
+> <!-- adds "translate-cloak" class while loading translations -->
+	<header><i></i></header>
+	<content>
+		<h2 class="dialog-title" translate>STARK.LOGIN.TITLE</h2>
+		<div *ngIf="userProfilesAvailable()">
+			<ul>
+				<li *ngFor="let user of users; trackBy: trackItemFn">
+					<a (click)="authenticateUser(user)">
+						<h3>{{ user.firstName }} {{ user.lastName }}</h3>
```

Comment:
`h3` add a margin causing a too big space between name and fonction, that should be removed.

Reply:
This will be tackled in #718